eventlet/__init__.py

```diff
@@ -17,7 +11,19 @@
 from eventlet import semaphore
 from eventlet import support
 from eventlet import timeout
+# NOTE(hberaud): Version are now managed by hatch and control version
+# hatch have a build hook who generate version file, however,
+# if the project is installed in editable mode then the _version.py file
+# will not be updated unless the package is reinstalled (or locally rebuilt).
+# for further details, please read:
+# https://github.com/ofek/hatch-vcs#build-hook
+# https://github.com/maresb/hatch-vcs-footgun-example
+try:
+    from eventlet._version import __version__
+except ImportError:
+    pass
```

Comment:
Maybe set it to some default value in this case? e.g. "0.0" or "unknown".

Reply:
Default to 0.0.0 but now that the path is fixed I don't think we will run the `except` part of this statement.